### PR TITLE
[DOC] Fix docstring of TransformerPipeline

### PR DESCRIPTION
**Pull request by aiwalter**
#### What does this implement/fix? Explain your changes.
Fix docstring of TransformerPipeline

### CODE REVIEW

**Comment by aiwalter** [comment]
looks good now

File: sktime/transformations/compose.py [hunk 1]
```diff
@@ -62,39 +62,42 @@ class TransformerPipeline(BaseTransformer, _HeterogenousMetaEstimator):
...
 
     The `TransformerPipeline` compositor allows to chain transformers.
-    The pipeline is constructed with a list of sktime transformers,
-        i.e., estimators following the BaseTransformer interface.
-    The list can be unnamed - a simple list of transformers -
-        or string named - a list of pairs of string, estimator.
+    The pipeline is constructed with a list of sktime transformers, i.e.
+    estimators following the BaseTransformer interface. The list can be
+    unnamed (a simple list of transformers) or string named (a list of
+    pairs of string, estimator.
```
Comment (fkiraly) [inline sktime/transformations/compose.py:68]: missing bracket

File: sktime/transformations/compose.py [hunk 2]
```diff
@@ -62,39 +62,42 @@ class TransformerPipeline(BaseTransformer, _HeterogenousMetaEstimator):
...
-        and returning the output of `trafoN.transform`
-    `inverse_transform` - result is of executing `trafo[i].inverse_transform`,
-        with `trafo[i].inverse_transform` input = output `trafo[i-1].inverse_transform`,
-        and returning the output of `trafoN.inverse_transform`
-    `update` - changes state by chaining `trafo1.update`, `trafo1.transform`,
-        `trafo2.update`, `trafo2.transform`, ..., `trafoN.update`,
-        where `trafo[i].update` and `trafo[i].transform` receive as input
+    the pipeline behaves as follows:
+
+        * `fit` - changes state by running `trafo1.fit_transform`,
```
Comment (fkiraly) [inline sktime/transformations/compose.py:73]: this renders strangly in readthedocs. Is the current rendering not ok?

File: sktime/transformations/compose.py [hunk 3]
```diff
@@ -62,39 +62,42 @@ class TransformerPipeline(BaseTransformer, _HeterogenousMetaEstimator):
...
+    inside the list of names leading up to it (inclusive)
 
-    `TransformerPipeline` can also be created by using the magic multiplication
-        on any transformer, i.e., any estimator inheriting from `BaseTransformer`
-            for instance, `my_trafo1 * my_trafo2 * my_trafo3`
-            will result in the same object as  obtained from the constructor
-            `TransformerPipeline([my_trafo1, my_trafo2, my_trafo3])`
-        magic multiplication can also be used with (str, transformer) pairs,
-            as long as one element in the chain is a transformer
+    The `TransformerPipeline` can also be created by using the magic multiplication
```
Comment (fkiraly) [inline sktime/transformations/compose.py:94]: should this be "a", not "the"?

File: sktime/transformations/compose.py (hunk 2 above)
Comment (aiwalter) [reply sktime/transformations/compose.py:73]: ther current rendering is weird bcz its showing your list on same level as the ``Parameters` and `Attributes` section:
image

**Review by fkiraly — APPROVED** [review]
well, I suppose it's an improvement. I would do it without the bullet points though, that looks odd. Can we not have the line with `fit`, `update` etc only printed in bold? Seems clearer than bullet point grey wall of text.

**Comment by aiwalter** [comment]
@fkiraly  pls check again, I removed this box. So lets get this in?

**Review by fkiraly — APPROVED** [review]
I approved already, and I think it looks even better now.

**Pull request merged**